config/eventing-istio/webhook/400-config-validation.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: config.webhook.eventing-istio.knative.dev
+  labels:
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/name: knative-eventing
+webhooks:
+- admissionReviewVersions: ["v1", "v1beta1"]
+  clientConfig:
+    service:
+      name: eventing-istio-webhook
+      namespace: knative-eventing
+  sideEffects: None
+  failurePolicy: Fail
+  name: config.webhook.eventing-istio.knative.dev
+  timeoutSeconds: 10
```

Comment:
I would add selectors here otherwise we get called on every CM

Reply:
see example:

https://github.com/knative-extensions/eventing-kafka-broker/blob/bd4ecfa772f15f6e7275c0946b9b806ff13c55fb/control-plane/config/eventing-kafka-broker/200-webhook/400-webhook-pod-defaulting.yaml#L34-L39